sdk/identity/Azure.Identity/src/DefaultAzureCredentialFactory.cs

```diff
@@ -279,10 +279,10 @@ public virtual TokenCredential CreateWorkloadIdentityCredential()
             return new WorkloadIdentityCredential(options);
         }
 
-        public virtual TokenCredential CreateManagedIdentityCredential()
+        public virtual TokenCredential CreateManagedIdentityCredential(bool isProbeEnabled = true)
         {
             var options = Options.Clone<DefaultAzureCredentialOptions>();
-            options.IsChainedCredential = true;
+            options.IsChainedCredential = isProbeEnabled;
```

Comment:
what does this option do?

Reply:
It determines whether the probe will be done and the retry behavior.